lib/pages/home/survey_carousel/survey_carousel_card.dart

```diff
@@ -22,49 +22,52 @@ class SurveyCarouselCard extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.start,
         mainAxisAlignment: MainAxisAlignment.end,
         children: [
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 24, vertical: 24),
-            child: Wrap(
-              children: [
-                Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  children: [
-                    Expanded(
-                      child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: [
-                          Text(
-                            surveyUiModel.title,
-                            style: Theme.of(context).textTheme.headline5,
-                            maxLines: 2,
-                            overflow: TextOverflow.ellipsis,
-                          ),
-                          Text(
-                            surveyUiModel.description,
-                            style: Theme.of(context).textTheme.bodyText1,
-                            maxLines: 2,
-                            overflow: TextOverflow.ellipsis,
-                          ),
-                        ],
+          Container(
+            color: Colors.black38,
```

Comment:
I added this small color box to make the text looks more readable (minor UI/UX update) 🙋